pybamm/experiment/experiment.py

```diff
@@ -42,13 +42,17 @@ class Experiment:
 
     def __init__(
         self,
-        operating_conditions,
-        period="1 minute",
-        temperature=None,
-        termination=None,
+        operating_conditions: list,
+        period: str = "1 minute",
+        temperature: float = None,
+        termination: list = None,
         drive_cycles=None,
         cccv_handling=None,
     ):
+        if not (isinstance(operating_conditions, list)):
+            raise TypeError("operating_conditions must be list of strings. For example:"
+                            f"\n\n [{operating_conditions}]")
+
```

Comment:
This is not needed. We document the type being list[str]. We don't check the types of the other arguments, so we should not check this one either.
```suggestion
```

Reply:
Also I think we should probably work to remove the blocks like
```python
if cccv_handling is not None:
            raise ValueError(
                "cccv_handling has been deprecated, use "
                "`pybamm.step.cccv_ode(current, voltage)` instead to produce the "
                "same behavior as the old `cccv_handling='ode'`"
            )
```
in a future change. If we only check them to warn that they are deprecated, then we should probably clean it up.

Reply:
Additionally if we remove the error handling it is less unit tests needed. For proper coverage you need to cover all error handling. So if we reduce it we need less tests.